assets/sass/components/tour-tooltip/_googlesitekit-tour-tooltip.scss

```diff
@@ -184,3 +184,25 @@
 		padding: 0 16px;
 	}
 }
+
+// Fix AdminMenuTooltip jumping on scroll by using fixed positioning.
+#react-joyride-step-0:has(.googlesitekit-tour-tooltip__fixed-settings-tooltip) > div.__floater {
+
+	left: 170px !important;
+	position: fixed !important;
+	top: 113px !important;
+	transform: none !important;
+}
+.folded {
+	#react-joyride-step-0:has(.googlesitekit-tour-tooltip__fixed-settings-tooltip) > div.__floater {
+
+		left: 46px !important;
+		top: 46px !important;
+	}
+}
+
+// Fix the arrow alignment to make it consistent to allow fixed positioning to align reliably.
+#react-joyride-step-0:has(.googlesitekit-tour-tooltip__fixed-settings-tooltip) div.__floater__arrow > span {
+	margin-top: 42px !important;
+	top: 0 !important;
+}
```

Comment:
Let's add comments explaining why we're using `!important` modifiers.

My guess here is that we need to override inline styles being supplied by Joyride, but we should be explicit about why we need to do that here since otherwise `!important` flags can be hard to debug/remove later.

Also: presumably these are set inline because Joyride moves these values around based on calculations/positioning. We should probably outline testing on a lot of viewports in the QA brief to ensure this doesn't break anything 🤔

Reply:
Thanks @tofumatt I updated this comment and extended the QAB